spec/soapy_bing/ads/bulk/parsers/csv_parser_spec.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+require 'json'
```

Comment:
I like the requires in files that actually uses the lib. Also, I just followed the convention that was already used in the project. If we decided to change, could be done in a different PR.